src/python-bindings/htcondor2/schedd.cpp

```diff
@@ -625,7 +625,8 @@ _schedd_submit( PyObject *, PyObject * args ) {
     }
 
     // Initialize the new cluster ad.
-    if( sb->init_base_ad( time(NULL), schedd.getOwner().c_str() ) != 0 ) {
+    auto_free_ptr owner(my_username());
+    if( sb->init_base_ad( time(NULL), owner ) != 0 ) {
```

Comment:
I don't have the faintest idea what the semantic effect of this change might be.  Is it even legal to set the cluster owner to anything other than the authenticated identity?

Reply:
I'm making this call site match all of the others. But it seems like all of them should be passing NULL for owner. You can set a non-matching owner if you authenticate as a queue superuser.

Reply:
The value being passed in is ignored if `SUBMIT_SHOULD_SET_LOCAL_OWNER` is `false`, which it's defaulted to since 8.9.5.  (This is news to me.)  I haven't the faintest idea how `Owner` is actually set when a queue superuser does it, but it does make sense to me that the `Owner` attribute come from the security layer, rather than the client (at least, in all other cases).

This might be a question to ask TJ -- the right thing to do might be to remove this argument entirely.

Reply:
someday we should support set_effective_owner on the schedd object for qmgmt calls from thje bindings, but until then, we should just init_base_ad with null.